### PR TITLE
binary search tree implemented

### PR DESCRIPTION
**Pull request by vikaskumr**

### CODE REVIEW

**Review by animeshk874 — APPROVED** [review]

**Comment by animeshk874** [comment]
@surya76657 can you review this?

File: public/data/data-structure-information/binary-search-tree.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          "label": "Binary Search Tree"
+        }
+      ],
+      "otherParameters": []
+    },
+    {
+      "key": "contains",
+      "methodName": "contains(value)",
+      "description": "Searches the value in binary search tree.",
+      "implementationCode": "contains(value) {\n\n        if (!value) {\n            throw new Error('Please enter the value to search!');\n        }\n\n        let currentNode = this;\n        while (currentNode !== null) {\n            if (value < currentNode.value) {\n                currentNode = currentNode.left;\n            } else if (value > currentNode.value) {\n                currentNode = currentNode.right;\n            } else if (value == currentNode.value) {\n                return true;\n            }\n        }\n        return false;\n    }",
```
Comment (surya76657) [inline public/data/data-structure-information/binary-search-tree.json]: Should be If, else if and else

File: public/data/data-structure-information/binary-search-tree.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "dependencies": ["getMinValue"],
+      "sources": [
+        {
+          "url": "https://www.geeksforgeeks.org/binary-search-tree-data-structure/",
+          "label": "Binary Search Tree"
+        }
+      ],
+      "otherParameters": []
+    },
+    {
```
Comment (surya76657) [inline public/data/data-structure-information/binary-search-tree.json:69]: This is not standard implementation of bst

**Review by surya76657 — APPROVED** [review]

**Comment by surya76657** [comment]
> @surya76657 can you review this?

Done

**Pull request merged**